kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -126,25 +127,26 @@ private void updateStubs() {
     ScopedSpan span =
         Tracing.currentTracer().startScopedSpan("KaldbDistributedQueryService.distributedSearch");
 
-    try {
-      List<ListenableFuture<SearchResult<LogMessage>>> queryServers = new ArrayList<>(stubs.size());
-      span.tag("queryServerCount", String.valueOf(stubs.size()));
-
-      for (KaldbServiceGrpc.KaldbServiceFutureStub stub : stubs.values()) {
-        ListenableFuture<KaldbSearch.SearchResult> searchRequest =
-            stub.withDeadlineAfter(READ_TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .withInterceptors(
-                    GrpcTracing.newBuilder(Tracing.current()).build().newClientInterceptor())
-                .search(request);
-        Function<KaldbSearch.SearchResult, SearchResult<LogMessage>> searchRequestTransform =
-            SearchResultUtils::fromSearchResultProtoOrEmpty;
-        queryServers.add(
-            Futures.transform(
-                searchRequest, searchRequestTransform::apply, MoreExecutors.directExecutor()));
-      }
+    List<ListenableFuture<SearchResult<LogMessage>>> queryServers = new ArrayList<>(stubs.size());
```

Comment:
Do the unit tests test these time outs?

Reply:
Yes, in `KaldbDistributedQueryServiceTest.testSearchWithOneShardTimeout`